### ISSUE TITLE
90s Timeout is (sometimes) too short for downloading vscode linux server

### ISSUE THREAD

**Issue by grapland0** [microsoft/vscode-remote-release#1733]
At the first time launching vscode with WSL, it downloads the linux server package in WSL. Such process is cancelled if the download cannot be finished in 90s.

Log:

```
[2019-10-25 22:59:44.902] Starting VS Code Server inside WSL (u1804)
[2019-10-25 22:59:44.903] Extension version: 0.39.5, Windows build: 18362. Multi distro support: enabled. WSL path support: enabled
[2019-10-25 22:59:45.206] Probing if server is already installed: C:\WINDOWS\System32\wsl.exe -d u1804 -e sh -c "[ -d ~/.vscode-server/bin/6ab598523be7a800d7f3eb4d92d7ab9a66069390 ] && echo found || echo missing"
[2019-10-25 22:59:45.496] No server install found on WSL, downloading server on client side: C:\Users\grapl\AppData\Local\Temp\vscode-remote-wsl\6ab598523be7a800d7f3eb4d92d7ab9a66069390\vscode-server-linux-x64.tar.gz
[2019-10-25 23:01:15.962] VS Code Server for WSL failed to start. No messages received for 90s
[2019-10-25 23:01:15.962] For help with startup problems, go to
[2019-10-25 23:01:15.963] https://code.visualstudio.com/docs/remote/troubleshooting#_wsl-tips
```

This binary is now ~30M. If we have 90s timeout, users must have a internet connection faster than 2.7Mbps, which is a little luxury for some emerging region.

**Comment by aeschli** [comment]
The timer is restarted after every message, such as a progress message. But yes, I can increase the limit.

**Comment by grapland0** [comment]
Looks good. Thank you!

**Comment by aeschli** [comment]
I pushed a fix for Remote-WSL 0.41.0 (requires vscode-insiders 1.41)

**Comment by aeschli** [comment]
dup of #1655
